examples/universal-with-router/common/actions/index.js

```diff
@@ -0,0 +1,42 @@
+export const SET_COUNTER = 'SET_COUNTER'
+export const INCREMENT_COUNTER = 'INCREMENT_COUNTER'
+export const DECREMENT_COUNTER = 'DECREMENT_COUNTER'
+
+export function set(value) {
```

Comment:
Let’s kill all of this with fire. I don’t think the “counter” example makes any sense for the universal example. Instead we should copy the components and logic form `async` and fetch *that* in the new (supposedly renamed) `async-universal`.